package.json

```diff
@@ -51,7 +51,7 @@
     "post-me": "^0.4.5",
     "punycode": "^2.1.1",
     "randombytes": "^2.1.0",
-    "skynet-js": "^4.0.20-beta",
+    "skynet-js": "^4.0.21-beta",
```

Comment:
We should bump this to the latest version and deal with the breaks in compatibility.

Reply:
Yeah, I'll upgrade to the latest version as part of the auto-login changes, after the corresponding changes in `skynet-js` are published.